components/useXChat/demo/stream-cancel.tsx

```
@@ -0,0 +1,119 @@
+
+  const abortRef = useRef(() => {});
+
+  useEffect(() => {
+    return () => {
+      abortRef.current();
+    };
+  }, []);
+
+  // Agent for request
+  const [agent] = useXAgent({
+    request: async ({}, { onSuccess, onUpdate }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)